app-backend/http4s-util/src/main/scala/com/rasterfoundry/http4s/TracedHTTPRoutes.scala

```diff
@@ -32,34 +32,49 @@ object TracedHTTPRoutes {
   ): Kleisli[OptionT[F, ?], AuthedRequest[F, User], Response[F]] = {
     Kleisli[OptionT[F, ?], AuthedRequest[F, User], Response[F]] { authedReq =>
       val req = authedReq.req
-      val operationName = "http4s-incoming-request"
-      val traceHeader =
+      val operationName = "http4s-request"
+      val tags = Map(
+        "http_method" -> req.method.name,
+        "request_url" -> req.uri.path.toString,
+        "environment" -> Config.environment
```

Comment:
Using the environment to make it easier to separate out traces for each deployment target